assets/js/components/ArticleDownloadLinksList.js

```diff
@@ -14,12 +14,16 @@ module.exports = class ArticleDownloadLinksList {
     this.doc = doc;
     this.$elm = $elm;
 
-    this.$sideSectionWrapper = this.doc.querySelector('.side-section-wrapper__list');
-    if (!this.$sideSectionWrapper) {
+    this.$buttonWrapper = this.doc.querySelector('.side-section-wrapper__list') ||
+        this.doc.querySelector('.button-collection--inline');
```

Comment:
I think we can use the id here rather than add a new class. @viktoria-mahmud @nlisgo what do you think?